kedro/templates/project/{{ cookiecutter.repo_name }}/tests/test_run.py

```diff
@@ -1,42 +1,20 @@
 """
-This module contains an example test.
-
+This module contains example tests for a Kedro project.
 Tests should be placed in ``src/tests``, in modules that mirror your
-project's structure, and in files named test_*.py. They are simply functions
-named ``test_*`` which test a unit of logic.
-
-To run the tests, run ``pytest`` from the project root directory.
+project's structure, and in files named test_*.py.
 """
-
 from pathlib import Path
 
-import pytest
-
-from kedro.config import OmegaConfigLoader
-from kedro.framework.context import KedroContext
-from kedro.framework.hooks import _create_hook_manager
-from kedro.framework.project import settings
-
-
-@pytest.fixture
-def config_loader():
-    return OmegaConfigLoader(conf_source=str(Path.cwd() / settings.CONF_SOURCE))
-
-
-@pytest.fixture
-def project_context(config_loader):
-    return KedroContext(
-        package_name="{{ cookiecutter.python_package }}",
-        project_path=Path.cwd(),
-        env="local",
-        config_loader=config_loader,
-        hook_manager=_create_hook_manager(),
-    )
-
+from kedro.framework.session import KedroSession
+from kedro.framework.startup import bootstrap_project
 
 # The tests below are here for the demonstration purpose
 # and should be replaced with the ones testing the project
 # functionality
-class TestProjectContext:
-    def test_project_path(self, project_context):
-        assert project_context.project_path == Path.cwd()
+
+class TestKedroRun:
+    def test_kedro_run(self):
+        bootstrap_project(Path.cwd())
```

Comment:
I think it would be helpful for users to explain what's happening here. Adding a link to the docs in the comments could provide more clarity:
[Testing a Kedro Project](https://docs.kedro.org/en/stable/tutorial/test_a_project.html).

It might also be a good idea to update the docs to include a more detailed explanation of this testing example.